Cargo.toml

```diff
@@ -15,28 +15,28 @@ license = "Apache-2.0"
 license-file = "LICENSE"
 
 [workspace.dependencies]
-blockifier = { git = "https://github.com/starkware-libs/blockifier.git", rev = "bf38764" }
-starknet_api = "0.3.0"
-cairo-lang-casm = "2.1.0-rc2"
-cairo-lang-runner = "2.1.0-rc2"
-cairo-lang-test-runner = "2.1.0-rc2"
-cairo-lang-sierra-to-casm = "2.1.0-rc2"
-cairo-lang-sierra = "2.1.0-rc2"
-cairo-lang-utils = "2.1.0-rc2"
-cairo-lang-starknet = "2.1.0-rc2"
-cairo-lang-compiler = "2.1.0-rc2"
-cairo-lang-filesystem = "2.1.0-rc2"
-cairo-lang-debug = "2.1.0-rc2"
-cairo-lang-defs = "2.1.0-rc2"
-cairo-lang-diagnostics = "2.1.0-rc2"
-cairo-lang-lowering = "2.1.0-rc2"
-cairo-lang-plugins = "2.1.0-rc2"
-cairo-lang-project = "2.1.0-rc2"
-cairo-lang-semantic = "2.1.0-rc2"
-cairo-lang-sierra-ap-change = "2.1.0-rc2"
-cairo-lang-sierra-gas = "2.1.0-rc2"
-cairo-lang-sierra-generator = "2.1.0-rc2"
-cairo-lang-syntax = "2.1.0-rc2"
+blockifier = { git = "https://github.com/starkware-libs/blockifier.git", rev = "df23f46" }
```

Comment:
Shouldn't we use a commit marked in a release tag? Or we just don't care about that

Reply:
They don't have a release tag for 2.1.0 afaik